src/Entities/ProductsEntity.php

```diff
@@ -0,0 +1,24 @@
+<?php
+class ProductsEntity
```

Comment:
This entity is for _a Product_, not _a Product**s**_

Also *some* white space would be nice.